translations/README.bn.md

```diff
@@ -8,6 +8,10 @@
 [![GitHub forks](https://img.shields.io/github/forks/microsoft/IoT-For-Beginners.svg?style=social&label=Fork)](https://GitHub.com/microsoft/IoT-For-Beginners/network/)
 [![GitHub stars](https://img.shields.io/github/stars/microsoft/IoT-For-Beginners.svg?style=social&label=Star)](https://GitHub.com/microsoft/IoT-For-Beginners/stargazers/)
 
+[![English](https://img.shields.io/badge/-English-red)](README.md)
+[![Chinese](https://img.shields.io/badge/-Chinese-yellow)](translations/README.zh-cn.md)
```

Comment:
This file is already in the translations folder, so the links to the translated versions don't need the path, just the file name.

Reply:
Oh yeah, sorry about that will fix it